src/client/pages/OfferNew/Perils/AmountCollection/AmountItem.tsx

```diff
@@ -58,21 +57,6 @@ const Value = styled.p`
   }
 `
 
-const StyledInfoIcon = styled(InfoIcon)`
```

Comment:
While checking for references to `InfoIcon` I found this one that it seems not to be used anywhere else, that's why the removal. Can you guys double check that?

Reply:
Good catch, that can be removed 🙌